saleor_app_framework/app_framework/schemas/manifest.py

```diff
@@ -0,0 +1,18 @@
+from typing import List
+
+from pydantic import BaseModel
+
+
+class Manifest(BaseModel):
+    name: str
+    version: str
+    about: str
+    dataPrivacy: str
+    dataPrivacyUrl: str
```

Comment:
Minor thing but I think we could use `snake_case` in manifest.json and as a result, in Python classes. It's just a JSON so the convention is up to us I believe. But this is a cosmetic change only, not an issue for now probably.

Reply:
I have to not agree here. 
- we already use the app interface starting from 2.11 (?). On the Saleor side, it already uses camelCase fields.
- camelCase is a recommendation for json structure. We don't build an interface (on saleor side) only for python apps.

Reply:
I see it is recommended by [Google ](https://google.github.io/styleguide/jsoncstyleguide.xml?showone=Property_Name_Format#Property_Name_Format) but don't see any information about JSON spec enforcing any of the formating. camelCase is fine, snake_case would be just more common with our backend stack.